phase1/gce/gce.jsonnet

```diff
@@ -157,11 +158,12 @@ function(cfg)
             "k8s-kubeadm-cni-plugin": if std.objectHas(p3, "cni") then p3.cni else "",
           } + if p2.provider == "kubeadm" then {
             "k8s-kubeadm-token": "${var.kubeadm_token}",
-            "k8s-kubeproxy-mode": "%(proxy_mode)s" % p2,
+            "k8s-kubeproxy-mode": if std.objectHas(p2, "proxy_mode") then "%(proxy_mode)s" % p2 else "",
```

Comment:
proxy-mode was added before my commit. But I found an issue if test-infra and kubernetes-anywhere are not syncrhonized. Adding a 'compatibility' layer.